ui/packages/catalog-ui-search/src/main/webapp/react-component/utils/validation/validation.tsx

```diff
@@ -16,7 +16,33 @@
 import { InvalidSearchFormMessage } from '../../../component/announcement/CommonMessages'
 const announcement = require('../../../component/announcement/index.jsx')
 
-export function validate(errors: any) {
+export function validate(filters: any, errors: any) {
```

Comment:
:exclamation: You now have method calls from @cassandrabailey293's commit that don't use this updated method signature